project/Dependencies.scala

```diff
@@ -15,7 +15,7 @@ object Dependencies {
   lazy val java8CompatVersion = settingKey[String]("The version of scala-java8-compat to use.")
 
   val junitVersion = "4.12"
-  val slf4jVersion = "1.7.25"
+  val slf4jVersion = "1.7.27"
```

Comment:
Hmm, I noticed there's an 1.8.0 beta and even a 2.0.0 alpha coming up... I wonder in how far they'll be binary compatible (https://www.slf4j.org/faq.html#changesInVersion18). They seem to be careful about this though.

A quick test suggests at least 2.0.0 isn't fully binary compatible (it lacks `getMarker`), but perhaps still 'good enough' that we could eventually consider updating to it.

Reply:
Let's look into that separately, but I don't think we have to be on any bleeding edge for these dependencies.